mpl.targets

```diff
@@ -37,19 +37,19 @@
       <!-- Remove files that are excluded from build -->
       <MPL Condition="%(MPL.ExcludedFromBuild) == true" Remove="@(MPL)"/>
 
-      <!-- Replace output file for files without partial compilation -->
-      <MPL Condition="%(MPL.PartialCompilation) != true" OutputFile="$(IntDir)mpl.ll"/>
+      <MPL Condition="%(MPL.CompilationMode) != Regular" OutputFile="$(IntDir)$([System.String]::Copy(%(Identity)).Replace(&quot;\&quot;, &quot;_&quot;)).ll"/> <!-- Replace output file for files without regular compilation -->
```

Comment:
$Identity preserves .mpl extension, do we really want it? Also, it seems we'll have problems if we use absolute paths here, is it ok?